extensions/smallrye-openapi/deployment/src/test/java/io/quarkus/smallrye/openapi/test/OpenApiResource.java

```diff
@@ -2,12 +2,18 @@
 
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
 
 @Path("/resource")
 public class OpenApiResource {
 
     @GET
-    public String root() {
-        return "resource";
+    public String root(@QueryParam("query") Query query) {
```

Comment:
Not a blocker but I think I would keep the initial method and add a new one named 'testEnums'. That would be clearer.

Reply:
Done  - I still had the IDE opened to the same file.